src/site/_data/vars.js

```diff
@@ -1,10 +1,11 @@
 const contactEmail = 'design-system@digital.mod.uk'
 
 module.exports = {
-  contactEmail,
   baseUrl: process.env.BASE_URL
     ? process.env.BASE_URL.replaceAll(/\/$/g, '')
     : 'https://design-system.service.mod.gov.uk',
+  contactEmail,
   markdownContactEmailLink: `[${contactEmail}](mailto:${contactEmail})`,
   noIndex: Boolean(process.env.NO_INDEX),
+  figmaLink: 'https://www.figma.com/file/JrmVrSdKCHUZBQFEB3xsjJ/Design-Library?node-id=50%3A5303&t=xG6UMcIDeLIbH1kD-1',
```

Comment:
If we moved `contactEmail` to make it alphabetical then this should go up a few lines :smile: